packages/app-admin/src/base/Base.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { memo } from "react";
+import { Extensions } from "@webiny/app-admin-core";
+import { AddMenu } from "~/index";
+import { plugins } from "@webiny/plugins";
+import { ReactComponent as DocsIcon } from "~/assets/icons/icon-documentation.svg";
+import { ReactComponent as SlackIcon } from "~/assets/icons/slack-logo.svg";
+import { ReactComponent as GithubIcon } from "~/assets/icons/github-brands.svg";
+import { ReactComponent as FileIcon } from "~/assets/icons/insert_drive_file-24px.svg";
+import { ReactComponent as SettingsIcon } from "~/assets/icons/round-settings-24px.svg";
+import { FileManager } from "~/components";
+
+import adminPlugins from "../plugins";
+
+const BaseExtension = () => {
+    plugins.register(adminPlugins());
```

Comment:
This is backwards compatibility-related?

Reply:
Yes, we'll be using this technique to hide plugins from user's projects, but still use them internally.